src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -209,7 +209,8 @@ public void testBCFIndex() {
         checkIndex(index, Arrays.asList("1"));
     }
 
-    @Test(expectedExceptions = UserException.CouldNotIndexFile.class)
+    // test disabled until https://github.com/samtools/htsjdk/issues/1323 is resolved
```

Comment:
Do we have a GATK issue to track this, and remind us to re-enable this test in the future?

Reply:
https://github.com/broadinstitute/gatk/issues/5838